packages/bot-web-ui/src/stores/load-modal-store.ts

```diff
@@ -14,7 +14,7 @@ interface ILoadModalStore {
     is_open_button_loading: boolean;
     is_strategy_loaded: boolean;
     loaded_local_file: File | null;
-    recent_strategies: string[];
+    recent_strategies: TStrategy[];
```

Comment:
I've changed the type of **recent_strategies** since we have in reality different:
<img width="1728" alt="Screenshot 2023-12-18 at 16 36 15" src="https://github.com/binary-com/deriv-app/assets/103181650/a09a0a59-2a10-4e59-92f4-b667fd23893a">